exec/jdbc/src/test/java/org/apache/drill/jdbc/test/Drill2489CallsAfterCloseThrowExceptionsTest.java

```diff
@@ -537,8 +537,6 @@ public void testclosedPreparedStmtOfOpenConnMethodsThrowRight() {
         new ClosedPreparedStatementChecker(PreparedStatement.class,
                                            closedPreparedStmtOfOpenConn);
 
-    checker.testAllMethods();
```

Comment:
Does deletion of the line means that test now checks nothing ? Maybe it's accidental change and needs revert or test method may be safely removed ?

Reply:
You're right. It was an accidental change, which is why it went through the tests. I'll restore it. Thanks for catching it.